code/orders-domain/pom.xml

```diff
@@ -8,5 +8,13 @@
   </parent>
 
   <artifactId>orders-domain</artifactId>
+    <dependencies>
```

Comment:
We cannot validate in our domain. I think the validation part must be on api and only validated data can be passed into the hexagon.

Reply:
Fixed